FirebaseAdmin/FirebaseAdmin/Messaging/AndroidNotification.cs

```diff
@@ -118,6 +172,265 @@ public sealed class AndroidNotification
         [JsonProperty("channel_id")]
         public string ChannelId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the "ticker" text which is sent to accessibility services. Prior to API level 21
+        /// (Lollipop), gets or sets the text that is displayed in the status bar when the notification
+        /// first arrives.
+        /// </summary>
+        [JsonProperty("ticker")]
+        public string Ticker { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the notification is automatically dismissed
+        /// or persists when the user clicks it in the panel. When set to false,
+        /// the notification is automatically dismissed. When set to true, the notification persists.
+        /// </summary>
+        [JsonProperty("sticky")]
+        public bool Sticky { get; set; }
+
+        /// <summary>
+        /// Gets or sets the time that the event in the notification occurred for notifications
+        /// that inform users about events with an absolute time reference. Notifications in the panel
+        /// are sorted by this time.
+        /// </summary>
+        [JsonIgnore]
+        public DateTime EventTimestamp { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not this notification is relevant only to
+        /// the current device. Some notifications can be bridged to other devices for remote display,
+        /// such as a Wear OS watch. This hint can be set to recommend this notification not be bridged.
+        /// See <a href="https://developer.android.com/training/wearables/notifications/bridger#existing-method-of-preventing-bridging">Wear OS guides</a>.
+        /// </summary>
+        [JsonProperty("local_only")]
+        public bool LocalOnly { get; set; }
+
+        /// <summary>
+        /// Gets or sets the relative priority for this notification. Priority is an indication of how much of
+        /// the user's attention should be consumed by this notification. Low-priority notifications
+        /// may be hidden from the user in certain situations, while the user might be interrupted
+        /// for a higher-priority notification.
+        /// </summary>
+        [JsonIgnore]
+        public AndroidNotification.PriorityType? Priority { get; set; }
+
+        /// <summary>
+        /// Gets or sets a list of vibration timings in milliseconds in the array to use. The first value in the
+        /// array indicates the duration to wait before turning the vibrator on. The next value
+        /// indicates the duration to keep the vibrator on. Subsequent values alternate between
+        /// duration to turn the vibrator off and to turn the vibrator on. If <see cref="VibrateTimingsMillis"/> is set and
+        /// <see cref="DefaultVibrateTimings"/> is set to true, the default value is used instead of
+        /// the user-specified vibrate_timings. A duration in seconds with up to nine fractional digits,
```

Comment:
Is the underscore in "vibrate_timings" a typo?

Reply:
I think it should be in code font. We use `vibrate_timings` with back-ticks in Node and Java SDKs. Changed the comments to use a class reference here but let me know if we should just call it `vibrate timings` instead. Thanks!